src/amo/components/AddonReview/index.js

```diff
@@ -179,7 +200,7 @@ export class AddonReviewBase extends React.Component {
           <input
             className="AddonReview-submit"
             type="submit"
-            value={i18n.gettext('Submit review')}
+            value={i18n.gettext('Update review')}
```

Comment:
It's pretty much always an update whenever you're on this screen. You can only see this the first time by selecting a star rating beforehand.

Reply:
There's not an easy way. I would have to make the internal state of `RatingManager` more complex and that test suite doesn't test the click-star-then-edit flow in its entirety (due to react-test-utils limitations).

I'll just revert it back to `Submit review` for now since that makes sense in either case.